calculator.go

```diff
@@ -1,13 +1,91 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"math"
+)
+
+var (
+	// errNotEnoughNumbers is returned by functions that expects
+	// minimum of two arguments.
+	errNotEnoughNumbers = errors.New("minimum two numbers are required")
```

Comment:
Instead of pre-declaring these error variables at package level, it might be a little easier to read if we simply use `errors.New` directly at the point of the error return. (Even more importantly, we'll be able to include more specific information about the invalid data, using `fmt.Errorf`.)